include/mbgl/ios/MGLTypes.h

```diff
@@ -12,3 +12,8 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode)
     /** The map follows the user location and rotates when the heading changes. */
     MGLUserTrackingModeFollowWithHeading
 };
+
+typedef struct {
+    CLLocationCoordinate2D sw;
+    CLLocationCoordinate2D ne;
+} MGLMapBounds;
```

Comment:
I thought it'd be more confusing to relate it to `region`, especially if we do decide to tackle the translation layer per https://github.com/mapbox/mapbox-gl-native/issues/1092#issuecomment-108005806. But I do see how `bounds` could be equally confusing. `MGLCoordinateBounds`?